package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "kani-vscode-extension",
-  "version": "0.0.3",
+  "version": "0.0.2",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "kani-vscode-extension",
-      "version": "0.0.3",
+      "version": "0.0.2",
```

Comment:
Is this intended to revert the version number? 
If yes, we may want to have a separate PR for it.

Reply:
No, this is a mistake that is being corrected. It shouldnt have been 0.0.3 from the start so this is fixing it, but I can absolutely fix that in a seperate PR. Thanks for pointing that out.